packages/ui/src/ui/pages/navigation/modals/RemoteCopyModal.tsx

```diff
@@ -186,6 +185,23 @@ function RemoteCopyModal() {
                     name: 'pool',
                     type: 'pool',
                     caption: 'Pool',
+                    warning: (
+                        <>
+                            pool must have specified{' '}
+                            {docsUrl(
+                                makeLink(
+                                    UIFactory.docsUrls['operations:remote_copy'],
+                                    'limits',
+                                    true,
+                                ),
+                            )}
+                        </>
+                    ),
+                    visibilityCondition: {
```

Comment:
I think If the field is not marked as required we don't need this condition, do we?